src/Buffer.h

```diff
@@ -104,6 +104,9 @@ class Buffer {
 
     typedef T ElemType;
 
+    // This is virtual solely to make some of the Python bindings easier to manage.
```

Comment:
The comment is waffling a bit. Either clients are allowed to subclass `Halide::Buffer`, in which case this is simply the correct thing. Otherwise they aren't in which case we should declare the class `final`. Point being the comment should make it clear whether subclassing this is ok or not. (I'm presuming it is a supported thing as we're using it in the Python bindings.)

Reply:
Clarified comment.